sdk/mobile-center-updates/src/main/java/com/microsoft/azure/mobile/updates/Updates.java

```diff
@@ -374,6 +374,13 @@ private synchronized void cancelPreviousTasks() {
     private synchronized void resumeUpdateWorkflow() {
         if (mForegroundActivity != null && !mWorkflowCompleted && isInstanceEnabled()) {
 
+            /* Don't go any further if the app was installed from an app store. */
```

Comment:
That would not simplify the code, today we start checking only when app starts and then we have a set of conditions. If I start moving that, I have also to move the isInstanceEnabled() check etc.... I would keep it as is.